assets/html/scss/documenter/layout/_search.scss

```diff
@@ -1,69 +1,133 @@
-      color: $search-result-badge-color;
-      background-color: $search-result-badge-background-color;
-      border-radius: 0.6rem;
-    }
-    
-    .search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
-      color: $search-result-link-text-color;
-      background-color: $search-result-link-text-background-color;
-    }
-    
-    .search-result-highlight {
-      background-color: $search-result-highlight;
-      color: black;
-    }
-    
-    .search-divider {
-      border-bottom: 1px solid $border;
-    }
-    
-    .search-result-title {
-      color: $search-result-title-text-color;
-    }
-    
-    .w-100 {
-      width: 100%;
-    }
-    
-    .gap-2 {
-      gap: 0.5rem;
-    }
-    
-    .gap-4 {
-      gap: 1rem;
-    }
-  }
+/* Code from https://dylanatsmith.com/wrote/styling-the-kbd-element */
+kbd.search-modal-key-hints {
+  border-radius: 0.25rem;
+  border: 1px solid $search-modal-kbd-color;
+  box-shadow: 0 2px 0 1px $search-modal-kbd-color;
+  cursor: default;
+  font-size: 0.9rem;
+  line-height: 1.5;
+  min-width: 0.75rem;
+  text-align: center;
+  padding: 0.1rem 0.3rem;
+  position: relative;
+  top: -1px;
+}
+
+.search-min-width-50 {
+  min-width: 50%;
+}
+
+.search-min-height-100 {
+  min-height: 100%;
+}
+
+.search-modal-card-body {
+  max-height: 500px;
+}
+
+.search-result-link {
+  border-radius: 0.7em;
+  transition: all 300ms;
+}
+
+.search-result-link:hover, .search-result-link:focus {
+  background-color: $search-result-link-hover;
+}
+
+.search-result-link .property-search-result-badge {
+  transition: all 300ms;
+}
+
+.property-search-result-badge {
+  padding: 0.15em 0.5em;
+  font-size: 0.8em;
+  font-style: italic;
+  text-transform: none !important;
+  line-height: 1.5;
+  color: $search-result-badge-color;
+  background-color: $search-result-badge-background-color;
+  border-radius: 0.6rem;
+}
+
+.search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
+  color: $search-result-link-text-color;
+  background-color: $search-result-link-text-background-color;
+}
+
+.search-filter {
+  @extend .property-search-result-badge;
+
+  color: $search-filter-color;
+  background-color: $search-filter-background-color;
+  transition: all 300ms;
+}
+
+.search-filter:hover, .search-filter:focus {
+  color: $search-filter-color;
+}
+
+.search-filter-selected {
+  color: $search-modal-selected-filter-color;
+  background-color: $search-modal-selected-filter-background-color;
+}
+
+.search-filter-selected:hover, .search-filter-selected:focus {
+  color: $search-modal-selected-filter-color;
+}
+
+.search-result-highlight {
+  background-color: $search-result-highlight;
+  color: black;
+}
+
+.search-divider {
+  border-bottom: 1px solid $border;
+}
+
+.search-result-title {
+  width: 85%;
+  color: $search-result-title-text-color;
+}
+
+.search-result-code-title {
+  font-size: 15px;
```

Comment:
This applies to multiple places, but let's not use `px` units, but instead use `rem` (except for like borders, shadows, rounded corners and such). Among other things, that (should be) consistent with how the rest of SCSS is written.

Also, this should be the equivalent of `14px` (to be consistent with the main body). We can probably just replicate the `rem`s of the main body (there should be a variable for that).